src/pool/srv_pool.c

```diff
@@ -817,18 +817,24 @@ ds_pool_svc_create(const uuid_t pool_uuid, int ntargets, const char *group,
 	return rc;
 }
 
+/** Stop any local PS replica for \a pool_uuid. */
 int
-ds_pool_svc_destroy(const uuid_t pool_uuid, d_rank_list_t *svc_ranks)
+ds_pool_svc_stop(uuid_t pool_uuid)
 {
-	d_iov_t		psid;
-	int		rc;
+	d_iov_t	id;
+	int	rc;
 
-	d_iov_set(&psid, (void *)pool_uuid, sizeof(uuid_t));
-	rc = ds_rsvc_dist_stop(DS_RSVC_CLASS_POOL, &psid, svc_ranks,
-			       NULL /* excluded */, true /* destroy */);
-	if (rc != 0)
-		D_ERROR(DF_UUID": failed to destroy pool service: "DF_RC"\n",
+	d_iov_set(&id, pool_uuid, sizeof(uuid_t));
+
+	rc = ds_rsvc_stop(DS_RSVC_CLASS_POOL, &id, false /* destroy */);
```

Comment:
Referring to the false /* destroy */ argument, is it worth a comment to remind the code reader that (as noted in the commit message) " the RDB files shall be deleted along with other pool files." by the target destroy handler?

And I guess the reason to specify false here is for a performance reason (to skip the remove() call in rsvc/srv.c stop())?